compiler/compile-code/compile-code-piggy_extractor

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env jq-f
+# compile-code-plpy_extractor -- TODO Compiles plpy_extractors in the normalized JSON into code executable by the runner
+##
+.deepdive_ as $deepdive
+
+| $deepdive.execution.processes | to_entries[]
+| .value.name = .key | .value
+| select(.style == "plpy_extractor")
```

Comment:
Good catch!  I'm actually planning to drop these in a later commit: f5186538f47e5a1f5020ce2420f3ec49b30a5b11 so maybe it's not too critical.